### PR TITLE
Home section

### PR DESCRIPTION
**Pull request by andrzejkukuryk**
I hope whole home section is responsive.
There's a mess in scsses. My idea is clean up this, use mixins etc when final UI will be ready.

### CODE REVIEW

File: src/data/personalLinks.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { ReactComponent as LinkedInIcon } from "../assets/linkedin.svg";
+import { ReactComponent as GitHubIcon } from "../assets/github.svg";
+import { ReactComponent as CvIcon } from "../assets/cv.svg";
+
+export interface PersonalLinksInfo {
+  icon: React.FunctionComponent<
```
Comment (anna-morawska) [inline src/data/personalLinks.tsx:6]: is it nessesary to use this complex icon type? Maybe ReactNode would be enough?

File: src/scenes/home/personalLinks.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { personalLinks } from "../../data/personalLinks";
+import { PersonalLinksItem } from "./personalLinksItem";
+import styles from "./personalLinks.module.scss";
+
+export function PersonalLinks() {
+  const createList = () => {
```
Comment (anna-morawska) [inline src/scenes/home/personalLinks.tsx:7]: is the separate function necessary? i would put it inline, inside div tag

File: src/scenes/home/softSkillsList.tsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { SoftSkillsItem } from "./softSkillsItem";
+
+export function SoftSkillsList() {
+  const softSkills = ["patient", "ambitious", "fast learner"];
```
Comment (anna-morawska) [inline src/scenes/home/softSkillsList.tsx:5]: i would move softskilss outside of the component, and replace  createList function with inline code

File: src/scenes/home/stackList.tsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import styles from "./stackList.module.scss";
+import { stack } from "../../data/stackIcons";
+import { StackItem } from "./stackItem";
+
+export function StackList() {
+  const createList = () => {
```
Comment (anna-morawska) [inline src/scenes/home/stackList.tsx:7]: the same, inline :)

File: src/data/personalLinks.tsx (hunk 1 above)
Comment (andrzejkukuryk) [reply src/data/personalLinks.tsx:6]: I'm affraid it is. When type of icon is set to ReactNode TS throws an error.

File: src/scenes/home/personalLinks.tsx (hunk 2 above)
Comment (andrzejkukuryk) [reply src/scenes/home/personalLinks.tsx:7]: Done!

File: src/scenes/home/softSkillsList.tsx (hunk 3 above)
Comment (andrzejkukuryk) [reply src/scenes/home/softSkillsList.tsx:5]: Done.

File: src/scenes/home/stackList.tsx (hunk 4 above)
Comment (andrzejkukuryk) [reply src/scenes/home/stackList.tsx:7]: Done.

File: src/scenes/menu/menu.tsx [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import styles from "./menu.module.scss";
+import { MenuItem } from "./menuItem";
+import { sections } from "../../models/menu";
+
+export function Menu() {
+  const createList = () => {
```
[the remark this answers is not in the document]
Comment (andrzejkukuryk) [reply src/scenes/menu/menu.tsx:7]: Done.

**Review by anna-morawska — APPROVED** [review]

**Pull request merged**